scss/reset/_forms.scss

```diff
@@ -81,15 +81,6 @@ label {
 	}
 }
 
-.label--field {
```

Comment:
`label--field` is the class applied to `label` elements in `meetup-web-components`